Source/Core/Core/IOS/Network/KD/Mail/MailCommon.h

```diff
@@ -11,6 +11,11 @@ namespace IOS::HLE::NWC24::Mail
 {
 constexpr u32 MAIL_LIST_MAGIC = 0x57635466;  // WcTf
 
+constexpr u32 CalculateFileOffset(u32 index)
```

Comment:
Ah, I forgot our swap functions aren't constexpr, that's my bad, this should be inline as it was before, sorry about that.